Revert this change that was committed to fix an issue with systemd.

We want to keep the original behaviour, i.e use the first unused vt.

--- startx.cpp.orig	2018-03-10 02:46:03.000000000 +0100
+++ startx.cpp	2018-03-12 20:18:16.065493050 +0100
@@ -200,17 +200,6 @@
 if [ x"$server" = x ]; then
     server=$defaultserver
 
-#ifdef __linux__
-    XCOMM When starting the defaultserver start X on the current tty to avoid
-    XCOMM the startx session being seen as inactive:
-    XCOMM "https://bugzilla.redhat.com/show_bug.cgi?id=806491"
-    tty=$(tty)
-    if expr "$tty" : '/dev/tty[0-9][0-9]*$' > /dev/null; then
-        tty_num=$(echo "$tty" | grep -oE '[0-9]+$')
-        vtarg="vt$tty_num -keeptty"
-    fi
-#endif
-
     XCOMM For compatibility reasons, only use xserverrc if there were no server command line arguments
     if [ x"$serverargs" = x -a x"$display" = x ]; then
 	if [ -f "$userserverrc" ]; then
